docs/F-Refinements/portability.md

```diff
@@ -1,94 +1,99 @@
+Mature languages attract international standards. These standards promote uniformity across host platforms. Committees of compiler writers, academics, applications developers and representatives from various national organizations review, discuss, negotiate and reach consensus on which elements should be included in the standard definition. Elements that are excluded are called extensions. Compiler writers include their _extensions_ in their own implementations of the agreed standard.
 
-A standard definition enables developers to write programs with the expectation that they will compile and run on a wide variety of platforms that support the standard.  By avoiding the use of extensions, we increase the portability of our software.  We call standard-compliant software *highly* portable. 
+A standard definition enables developers to write programs with the expectation that they will compile and run on a wide variety of platforms that support the standard. By avoiding the use of extensions, we increase the portability of our software. We call standard-compliant software _highly_ portable.
 
-This chapter reviews the C standards that have been defined throughout the language's history, introduces some coding guidelines and describes utilities available for analyzing the degree of portability.
+This chapter reviews the C standards that have been defined throughout the language's history, introduces some coding guidelines, and describes utilities available for analyzing the degree of portability.
 
 ## The C Standards
 
-Dennis Ritchie created the C language between 1969 and 1973, while working at AT&T Bell Labs.  Brian Kernighan and Dennis Ritchie published their original description of the language in the classic book entitled "The C Programming Language" in 1978.  They published a second edition in 1988. 
+Dennis Ritchie created the C language between 1969 and 1973, while working at AT&T Bell Labs. Brian Kernighan and Dennis Ritchie published their original description of the language in the classic book entitled "The C Programming Language" in 1978. They published a second edition in 1988.
 
-Since then, the C language has undergone three 'standard' definitions: one in 1989, one in 1999 and the latest in 2011. 
+Since then, the C language has undergone three 'standard' definitions: one in 1989, one in 1999 and the latest in 2011.
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/standards.png)
+![standards](https://ict.senecacollege.ca//~ipc144/pages/images/standards.png)
```

Comment:
Could you have a more descriptive alt text? Something along the lines of "Evolution of the C standard over the years" would be fitting.

Reply:
Please either file a new Issue for this improvement (I agree with this) or make modifications now. Thanks!

Reply:
Hello @CameronGray1210, I have made all the changes based on the reviews of @JerryHue. Could you please take a look at it.